index.md

```diff
@@ -1,78 +1,47 @@
-- [Liquid staking](/concepts/staking/#liquid-staking) for capital efficiency and a [Penalty-Absorbing Stake](/concepts/staking/#penalty-absorbing-stake-pas) model. [Delegated](/glossary/#delegated) _Newton_ stake yields transferrable _Liquid Newton_ tokens, which can be used to redeem the underlying _Newton_ by unbonding stake. [Self-bonded](/glossary/#self-bonded) stake by validator operators does not yield _Liquid Newton_ and is slashed before delegated stake to assure validator integrity and 'skin in the game'.
-- [Accountability and fault detection (AFD)](/concepts/accountability/): a protocol for detecting infractions of consensus rules by validators participating in block proposal and voting as a committee member. Failure to adhere to these rules is a rule infraction and AFD will detect and apply slashing penalties for proven rule infractions.
-- [Oracle network](/concepts/oracle-network/): a protocol for consensus-computed median price data for selected FX currency pairs as an L1 platform feature. The submission of price data for aggregation and voting is a validator responsibility; validator nodes form an oracle network. 
+![](/_assets/images/text-logo-autonity.svg)
 
+Welcome to the documentation site for the Autonity Go Client (AGC) and Autonity Oracle Server (AOS). AGC is the main client software run by peer nodes in an Autonity network. AOS is the oracle software run by validator nodes to source price data from external data providers and submit price reports on-chain.
 
-## Technology
+AGC is a fork of [Geth <i class='fas fa-external-link-alt'></i>](https://geth.ethereum.org/) and is the reference implementation of the Autonity Protocol. The protocol provides an EVM-based blockchain that utilises Tendermint BFT for delegated proof of stake consensus, has a dual native coin design for tokenomics, and provides liquid staking for capital efficiency.
 
-Core technology is the [Autonity Go Client (AGC)](/concepts/client/), a fork of the Go Ethereum '[Geth](https://geth.ethereum.org/)' client, and the [Autonity Oracle Server (AOS)](/concepts/oracle-server/). For the Geth base version of AGC see Reference, [Codebase](/reference/codebase/).
+This documentation describes key concepts and functionality of the Autonity protocol, main client, and oracle server. It explains how to use the software and connect to an Autonity network.
 
-AGC is the reference implementation of the Autonity Protocol and provides the main client software run by [participant](/glossary/#participant) peer nodes in an Autonity network system - see [Running a Node](/node-operators/). AOS is operated by validator nodes only - see [Running a validator](/validators/).
+Autonity is an EVM based public blockchain for decentralized derivatives clearing.
```

Comment:
Not a big deal but I would maybe merge this sentence somehow with the 2nd paragraph?

To me it is a bit weird to have this sentence alone at the end of the "chapter"

Reply:
OK, I have removed this 'strapline' which is from autonity.org. Agree with you.